docs/pages/docs/handbook/testing.mdx

```diff
@@ -0,0 +1,123 @@
+import { Tabs, Tab } from '../../../components/Tabs'
+import Callout from '../../../components/Callout'
+
+# Testing in a Monorepo
+
+Along with linting and building, testing is a crucial part of a production-ready monorepo. Whether you're using end-to-end tests or a unit test suite, integrating them with Turborepo will lead to enormous speed-ups.
+
+## Working with test runners
+
+Let's say we have a monorepo that looks like this:
+
+```
+├── apps
+│   └── web
+│       └── package.json
+└── packages
+    └── shared
+        └── package.json
```

Comment:
nit: the tabbing here is different than other pages. also, we maybe should do tab characters instead of space characters?